hosts/default.nix

```diff
@@ -5,13 +5,14 @@
   config,
   pkgs,
   ...
-}: {
+}:
+{
   nixpkgs = {
     # You can add overlays here
     overlays = [
       # Add overlays your own flake exports (from overlays and pkgs dir):
       self.outputs.overlays.additions
-      self.outputs.overlays.modifications
+      # self.outputs.overlays.modifications
```

Comment:
why this?

Reply:
I commented the over modifications out since it wasn't doinging anything